scene/resources/2d/skeleton/skeleton_modification_2d_physicalbones.cpp

```diff
@@ -67,7 +67,9 @@ bool SkeletonModification2DPhysicalBones::_get(const StringName &p_path, Variant
 #ifdef TOOLS_ENABLED
 	if (Engine::get_singleton()->is_editor_hint()) {
 		if (path.begins_with("fetch_bones")) {
-			return true; // Do nothing!
+			// Do nothing!
+			r_ret = false;
+			return true;
```

Comment:
Not sure about this one. Why `r_ret = false`?

Reply:
Is it ok to override _set but not _get? I don't really know what the behavior is there.

But this boolean property is used as a button (you click it to trigger a function), so it should never have a true as a value. So this enforces it.

Reply:
> Is it ok to override _set but not _get?

Yes, trying to get property will just return null.
I'm not familiar with this code either, but the intention seems to have a property that can only be set.

Reply:
Right I guess there's not much difference between "retrieve the current value (which is always `false` due to setter doing nothing)" vs "always return false". I'll revert this change.

Reply:
Or the whole block could be left out, since there's a `return false` further down as well... I guess it's better to be more explicit and leave it in?